src/cli-options.ts

```diff
@@ -448,3 +451,47 @@ function parseRemainingArgs(
     extraArgs,
   };
 }
+
+const DEFAULT_WATCH_STRATEGY = {strategy: 'event'} as const;
+const DEFAULT_WATCH_INTERVAL = 500;
+
+/**
+ * Interpret the WIREIT_WATCH_* environment variables.
+ */
+function readWatchConfigFromEnv(): Options['watch'] {
+  switch (process.env['WIREIT_WATCH_STRATEGY']) {
+    case 'event':
```

Comment:
Style nit: I definitely prefer avoid fall-through myself. Would this be easier as an if/else statement?

Reply:
This whole file (and other files) uses this style of switching on `state`. We're doing exhaustive checking for every state for every transition. I actually kinda like using fallthrough here, it's just the most concise way to write out all the permutations.